docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -35,30 +35,26 @@ If the application resumes data successfully:
     * `SubscribeButton`
     * `SubscibeVehicleData`
 
-If the application does not resume data successfully:
+If the application does NOT resume data successfully:
 
   * SDL will provide `OnAppRegistered` with `resumeVrGrammars`:`false` or no resume parameter at all.
   * SDL cleans up all previously stored application data for the application that failed to resume. The HMI must also clean up previously compiled `VRGrammars` for the application.
   * The application will send new data to start SDL operations. In this event, SDL and the HMI should restart the cycle of collecting application data for resumption.
 
-!!! MUST
+_**HMI must:**_
 
-  1. HMI must update its list of registered applications.
-  2. HMI must store the application data sent in the `applications` parameter.
-  3. HMI must compile and store `VRGrammars` for the `vrSynonyms` parameter, and arrange them for the user to be able to use via voice recognition. Note: The VR commands to activate an application must be accessible when viewing a different active application or the list of registered applications.
-  4. HMI must provide the user with the possibility to choose an application among a list of registered applications.
-  5. HMI must send an `OnAppActivated` notification to SDL when the user activates an app via the `UI` or `VR`.
+  1. Update its list of registered applications.
+  2. Store the application data sent in the `applications` parameter.
+  3. Compile and store `VRGrammars` for the `vrSynonyms` parameter, and arrange them for the user to be able to use via voice recognition. Note: The VR commands to activate an application must be accessible when viewing a different active application or the list of registered applications.
+  4. Provide the user with the possibility to choose an application among a list of registered applications.
+  5. Send an `OnAppActivated` notification to SDL when the user activates an app via the `UI` or `VR`.   
+  6. Manage application events by priority. HMI gets proirity information from _OnAppRegistered_, _UpdateAppList_, _ActivateApp_ HMI API.
 
-!!!
-
-!!! NOTE
-
-If a device is connected over USB and registers an application, SDL will send `OnAppRegistered` with a hash of the usb serial number as the device id.
-
-If a device is connected over Bluetooth or Wi-Fi and registers an application, SDL will send `OnAppRegistered` with a hash of the device's mac address as the device id.
-
-!!!
+_**Note:**_   
 
+   - If a device is connected over USB and registers an application, SDL will send `OnAppRegistered` with a hash of the usb serial number as the device id.
+   - If a device is connected over Bluetooth or Wi-Fi and registers an application, SDL will send `OnAppRegistered` with a hash of the device's mac address as the device id.
+   - When the application is registered for the first time (no records in PT) PoliciesManager should not initiate prompting the User about the event.
```

Comment:
This section should use the 

```
!!! NOTE
Content
!!!
```

format as specified in the [SDL Markdown Spec](https://github.com/smartdevicelink/sdl_markdown_spec).